### PR TITLE
feat: a couple of UI improvements

### PR DESCRIPTION
**Pull request by fk**
- switch a couple of Font Awesome icons to their Material Design equivalents
- center icons in `FloatingButton`
- use the system font stack for the list views `from` and `subject`
- calmer color for list view card border bottom and `from`
- adjust header height
- add `Spinner` component and use it for "Loading mail…" and "Fetching message details…" loading messages
- a couple minor icon whitespace/margin and positioning fixes

### CODE REVIEW

**Comment by fk** [comment]
Ah dang, gimme a sec…

File: src/components/footer.js [hunk 1]
```diff
@@ -80,9 +81,8 @@ function Footer({ stripes }) {
                   <FaGithub
                     size={16}
                     css={{
-                      marginRight: '0.25rem',
-                      position: 'relative',
-                      top: 2,
+                      marginRight: '0.5rem',
+                      verticalAlign: 'sub',
```
Comment (DSchau) [inline src/components/footer.js]: 👍

File: src/components/footer.js [hunk 2]
```diff
@@ -92,10 +92,10 @@ function Footer({ stripes }) {
                 <Link
                   href={`${data.site.siteMetadata.repository.url}/stargazers`}
                 >
-                  <strong css={{ marginRight: 2 }}>
+                  <strong css={{ fontWeight: 'normal', marginRight: 2 }}>
```
Comment (DSchau) [inline src/components/footer.js]: heh may as well just use a span here, but nothing to not merge over.

File: src/components/footer.js (hunk 1 above)
Comment (fk) [reply src/components/footer.js]: It does what we want it to, but not sure why ;) 🤷‍♂️

Comment (DSchau) [reply src/components/footer.js]: It's _way_ cleaner than what I had 😆

Comment (DSchau) [reply src/components/footer.js]: Hate when I have to resort to those hacks!

**Pull request merged**